csrc/scheduler/matmul.cpp

```diff
@@ -717,6 +701,12 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
       {acr, bcr, ab, bb, a, b},
       {ParallelType::TIDy, ParallelType::TIDz});
 
+  // auto inline for all tensors except register tensors and output tensor
+  inlineMost(ir_utils::allTvsExcept(fusion, {acr, bcr, ab, bb, c}));
+
+  // if auto inline, will inline to position-7, leads to performance regression
+  inlineSelectedAt({acr, bcr, ab, bb}, {cc}, 6);
```

Comment:
novice question: why would `cc` need curly brackets?

Reply:
No, it is not needed. Thanks for reporting. I will write a quick fix right now.

Reply:
for the record https://github.com/NVIDIA/Fuser/pull/205